pkg/apis/pipeline/v1/task_types.go

```diff
@@ -119,6 +120,9 @@ type TaskSpec struct {
 	// Results are values that this Task can output
 	// +listType=atomic
 	Results []TaskResult `json:"results,omitempty"`
+
+	// PodTemplate holds pod specific configuration
+	PodTemplate *pod.PodTemplate `json:"podTemplate,omitempty"`
```

Comment:
Can you add an example to the examples directory? They act as e2e tests as well and help validate if the given code is working properly.

Reply:
Certainly!